src/test/java/edu/iis/powp/events/SelectLoadSecretCommandOptionListener.java

```diff
@@ -30,8 +31,10 @@ public void actionPerformed(ActionEvent e) {
 		commands.add(new DrawToCommand(70, 0));
 		commands.add(new DrawToCommand(70, 50));
 		commands.add(new DrawToCommand(20, 50));
+		
+		ComplexCommand complexCommand = new ComplexCommand(commands);

```

Comment:
FIX TO MERGE: Application (in general) should still rely on ICompoundCommand. Such modifications are not allowed. Others groups can introduce their own implementations of ComplexCommand and must have similar restrictions!